src/getMetroConfig.js

```diff
@@ -11,10 +11,10 @@ const dedent = require('dedent-js')
 const getDependencyPath = require('./getDependencyPath')
 
 const mapModule = name =>
-    `    '${name}': path.resolve(__dirname, 'node_modules/${name}')`
```

Comment:
Could you please revert these cosmetic changes, this actually pretty prints a config file with indentation and the change is unrelated to the prupose of this PR

Reply:
Sorry for adding the spacing changes. 

I figured that moving the indentation spacing to the `.join()` statements would be better which you'll see below. This only affects a few lines. Doing so also means that the indentation remains constant in the big templated string at the end of this file, which currently has inconsistent/jagged spacing. This still results in a properly formatted output file and restricts all extra spacing to the three join statements.

You can preview this PR without the whitespace by [appending `?w=1` to the GitHub URL](https://github.com/MrLoh/metro-with-symlinks/pull/9/files?w=1). They've also added a UI for that under the `Diff settings` near the `Review changes` button at the top right of the diff page.

If you'd still like me to back this out, let me know and I can do so.